sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -2523,7 +2523,8 @@ public async Task GetBlobClient_SpecialCharacters(string blobName)
             // Arrange
             await using DisposingContainer test = await GetTestContainerAsync();
 
-            Uri expectedUri = new Uri($"https://{TestConfigDefault.AccountName}.blob.core.windows.net/{test.Container.Name}/{Uri.EscapeDataString(blobName)}");
+            string endpointSuffix = Environment.GetEnvironmentVariable("STORAGE_ENDPOINT_SUFFIX") ?? Constants.ConnectionStrings.DefaultEndpointSuffix;
```

Comment:
Could you extract this to StorageTestBase (or whatever base class is) so that this isn't repeated across tests? 

Or create `static string` constant somewhere? assuming this doesn't change over lifespan of test run.

Reply:
Yeah, I would change it.